Misc/NEWS.d/next/Core and Builtins/2019-12-31-19-06-02.bpo-39171.cmXDsX.rst

```diff
@@ -0,0 +1 @@
+Creating missing parent in tkinter simpledialog when possible.
```

Comment:
Would this be better suited to the "library" part of the news?

Reply:
I don't know much about these aspects.